dev/release/verify-release-candidate.sh

```diff
@@ -566,6 +582,7 @@ test_package_java() {
   show_header "Build and test Java libraries"
 
   maybe_setup_conda maven openjdk
+  maybe_setup_maven
```

Comment:
Could you use `maybe_setup_maven` -> `maybe_setup_conda` order like others?

```suggestion
  maybe_setup_maven
  maybe_setup_conda maven openjdk
```